test/test_async_closures.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for xla_dist."""
+from __future__ import division
+from __future__ import print_function
```

Comment:
pytorch/xla supports minimum python version 3.6 I think, so it's probably safe to remove these `__future__` imports?
(python2 has been deprecated in pytorch releases as well :D)

Reply:
done